### DISCUSSION TITLE
Support automation groups/categories

### DISCUSSION THREAD

**Discussion by andreas-bulling** [home-assistant/frontend#19212] [Automations / scripts / scenes]
As a Home Assistant setup grows, so does the number of automations. While one can search for a specific automation, one has to remember at least part of its name.

It would be nice if there was the possibility to create "automation groups/categories" into which automations can be sorted, e.g. "Light Automations", "Cover Automations" etc. These groups don't have any functional meaning - they are just so that instead of one long list the user is presented with categories.

**Comment by ildar170975** [comment]
https://community.home-assistant.io/search?q=automations%20%23feature-requests%20folder
